website/addons/box/templates/log_templates.mako

```diff
@@ -1,6 +1,7 @@
 <script type="text/html" id="box_file_added">
 added file
-<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">{{ params.fullPath }}</a> to
+<a class="overflow log-file-link" data-bind="click: NodeActions.addonFileRedirect">
+    {{ params.fullPath.indexOf('/') == 0 ? params.fullPath.replace('/', '', 1) : params.fullPath }}</a> to
```

Comment:
Did using `String.replace` with a Regex--as suggested in my previous comment--not work correctly?